ics_space_amplification_test.py

```diff
@@ -2,12 +2,12 @@
 import time
```

Comment:
Changes to this file may temporarily be ignored until we confirm if there's still a need to keep it for backward comparisons

Reply:
Nothing is permanent as a temporary thing.

If it stays, need to be review like any other change.

If it goes away, let's delete it now. 

If you think it's irrelevant to this PR, undo the changes to it.